lib/Onyx.js

```diff
@@ -780,6 +780,39 @@ function mergeCollection(collectionKey, collection) {
         });
 }
 
+/**
+ * Insert API responses and lifecycle data into Onyx
+ *
+ * @param {Object} data An object containing the Onyx data to set
```

Comment:
This should be an `Array`, no?

```suggestion
 * @param {Array} data An array containing the Onyx data to set
```

Reply:
Good call! Thanks!